src/components_redux/Input.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+
+class Input extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.openNav = this.openNav.bind(this);
+    //this.closeNav = this.closeNav.bind(this);
+  }
+
+  componentDidMount() {
+    document.addEventListener("click", this.closeNav);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener("click", this.closeNav);
+  }
+
+  openNav() {
+    const style = { width: 550 };
+    this.setState({ style });
+    document.body.style.backgroundColor = "rgba(0,0,0,0.4)";
+    document.addEventListener("click", this.closeNav);
+  }
+
+
+  render() {
+    return (
+      <div>
+        <div className="input-conntainer">
+          <input
+            onClick={this.openNav}
+            className="input-field"
+            type="text"
+            placeholder="Click here to for input text"
+          ></input>
+        </div>
+        <div ref="snav" className="overlay" style={this.state.style}>
+          <h2>Fill the form</h2>
+          <label>
+            Some text:
+            <input type="text"></input>
+          </label>
+          <br></br>
```

Comment:
`br` (and all following br's) should be only use inside content and not for design.

Reply:
I tested something with it and forgot to remove it.